### PR TITLE
Bugfix/ensure option changes bubble up to select

### PR DESCRIPTION
**Pull request by mrcthms**
In Firefox, when using the validation library on a <select> element (notably on the Add Address screen on TJH), the `$input` variable is set to the `<option>` rather than the `<select>`, which throws an error when trying to get `$input.data()`. So, if the current element being validated is an `<option>` tag, we set it to be it's parent `<select>` instead.

### CODE REVIEW

File: src/js/validate.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ var validate = module.exports = validators.validate = {};
 validate.element = function validateElement(input, setClasses) {
   var $input = $(input);
   var validations;
+  var inputTag = $input.prop('tagName').toLowerCase();
+  if (inputTag === 'option') {
```
Comment (callumacrae) [inline src/js/validate.js]: `input.tagName === 'OPTION'` seems simpler

**Comment by callumacrae** [comment]
:+1:

File: src/js/validate.js (hunk 1 above)
Comment (mrcthms) [inline src/js/validate.js]: Fair point, updated.

**Comment by djgrant** [comment]
@mrcthms Great find! I feel like we should be doing this check in `addListeners.js` though. Maybe it can be resolved using `event.target` instead of `this`?

**Comment by mrcthms** [comment]
@djgrant Actually looks like 1.1.0 fixes this already, so no need to merge this.

**Pull request closed without merging**